### DISCUSSION TITLE
Protein ratios within a sample

### DISCUSSION THREAD

**Discussion by sofiakalaid** [vdemichev/DiaNN#744] [Q&A] [answered]
Dear Vadim,

I'm running a label-free experiment where I want to compare the ratio of certain proteins with high sequence similarity in each of my samples. I'm trying to figure out which of the quantification algorithms is more accurate in my case. 
MaxLFQ (if I'm correct) is calculated based on all precursor intensities and not just the proteotypic ones, which is not ideal for very homologous proteins. On the other hand, with the Top3 I risk that some proteins have less than 3 unique peptides identified. Should I then calculate the MaxLFQ using only the unique peptides? Or maybe use the Top1 precursor intensity or some other kind of values like iBAQ?

On another note, I have noticed in the main-report file that some non-proteotypic peptides are assigned to several Protein.Ids , but they have only one Protein.Group entry. Which Protein.Group entry is selected then and why don't I get all the entries that I see in the Protein.Ids column (DIA-NN v.1.8.1)?

![main-report-screenshot](https://github.com/vdemichev/DiaNN/assets/61777933/ee444485-50a2-4f87-bfbf-531bba658f50)

Thanks a lot in advance for your help and your effort on answering all the questions on GitHub!

Best regards,
Sofia

**Comment by vdemichev** [comment, marked as answer]
Hi Sofia,

You can always perform MaxLFQ (using iq or diann R package) on your peptides of choice (in this case proteotypic with the right proteotypicity definition) - this is what I would suggest.

Report column contents are defined in the DIA-NN docs.

Best,
Vadim

**Reply by sofiakalaid** [reply]
Hi Vadim,
Many thanks for your response!
Which proteotypicity definition would you recommend in this case? Should I extract the unique peptides at the isoform or protein level instead of gene level? 

Best,
Sofia

**Reply by vdemichev** [reply]
What kind of samples are you analysing?

**Reply by sofiakalaid** [reply]
It's human plasma and I'm using the Human Uniprot database with the canonical+isoforms entries.

**Reply by vdemichev** [reply]
I would suggest to keep it set to 'Genes', unless you are spefically interested in distinguishing isoforms - then set to isoforms.

**Reply by sofiakalaid** [reply]
Great, thanks a lot!
